Tasks/FileTransform/task.json

```diff
@@ -0,0 +1,72 @@
+{
+    "id": "8ce97e91-56cc-4743-bfab-9a9315be5f27",
+    "name": "FileTransform",
+    "friendlyName": "File Transform",
+    "description": "File Transform",
+    "author": "Microsoft Corporation",
+    "helpMarkDown": "File Transforms & Variable Substitution",
+    "category": "Utility",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "preview": "true",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "demands": [],
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "instanceNameFormat": "File Transform: $(Package)",
+    "inputs": [
+        {
+            "name": "folderPath",
+            "type": "filePath",
+            "label": "Folder Path",
+            "defaultValue": "$(System.DefaultWorkingDirectory)/**/*.zip",
+            "required": true,
+            "helpMarkDown": "File path to the package or a folder.<br />Variables ( [Build](https://docs.microsoft.com/vsts/pipelines/build/variables) | [Release](https://docs.microsoft.com/vsts/pipelines/release/variables#default-variables)), wildcards are supported. <br/> For example, $(System.DefaultWorkingDirectory)/\\*\\*/\\*.zip."
```

Comment:
File path to the "package" or a folder ? 🙀

Reply:
Will take in next PR